app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,35 @@
 class SurveysController < ApplicationController
+
   def send_survey
+    respondent_ids = JSON.parse(params[:respondent_ids]).uniq
+
+    begin
+      survey_id = params[:id]
+      survey = Survey.find(survey_id)
+    rescue ActiveRecord::RecordNotFound
+      return render :json => {:error => :survey_not_found}, :status => :bad_request
+    end
+
+    if survey.first_question.nil?
```

Comment:
Think this should be encapsulated within the survey model as `ready_to_send?` or something similar. I can see this conditional growing to include more requirements on the survey model. Don't think the error needs to be that specific; it's probably enough to say survey is not in a sendable state.